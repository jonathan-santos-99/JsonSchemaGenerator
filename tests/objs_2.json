{
  "level_1": {
    "level_2_1": 1,
    "level_2_2": "teste"
  }
}