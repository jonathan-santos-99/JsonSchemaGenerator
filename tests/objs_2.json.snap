{"$schema":"http://json-schema.org/draft-04/schema#","type":"object","properties":{"level_1":{"type":"object","properties":{"level_2_1":{"type":"integer"},"level_2_2":{"type":"string"}}}}}